capture-sdk/sdk/src/main/java/net/gini/android/capture/error/ErrorFragmentCompat.kt

```diff
@@ -18,7 +18,7 @@ import net.gini.android.capture.internal.util.AlertDialogHelperCompat
  * </p>
  * <p>
  * Your Activity must implement the {@link ImageRetakeOptionsListener} interface to receive events
- * from the No Results Fragment. Failing to do so will throw an exception.
+ * from the Error Fragment. Failing to do so will throw an exception.
```

Comment:
[Request] Please add `Internal use only.` to the beginning of this javadoc. We don't want clients to think that they can use our fragments directly. 🙅‍♂️